src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\ContentType\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+/**
+ * This is the class that validates and merges configuration from your app/config files.
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/configuration.html}
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('cmf_content_type');
+        $rootNode->children()
+            ->arrayNode('mapping')
+                ->useAttributeAsKey('class_fqn')
```

Comment:
Initially there is an Array metadata driver, the idea being that we can configure directly in the DI container, later we can replace with a chain driver implementation for YAML/XML/Annotation support.